src/main/java/seedu/address/logic/commands/recipe/RecipeAddStepCommand.java

```diff
@@ -23,6 +23,7 @@
 public class RecipeAddStepCommand extends RecipeAddCommand {
 
     public static final String MESSAGE_SUCCESS = "New step added for %1$s: %2$s";
+    public static final String NEXT_KEY_WORD = "next";
```

Comment:
I think between key and word don't need underscore

Reply:
fixed thanks!